ts/src/bitso.ts

```diff
@@ -1056,7 +1056,8 @@ export default class bitso extends Exchange {
     parseOrderStatus (status) {
         const statuses = {
             'partial-fill': 'open', // this is a common substitution in ccxt
-            'completed': 'closed',
+            'partially filled': 'open',
+            'completed': 'closed'
```

Comment:
Hello @mironsv,
Thanks for your contribution, I think we can also add `queued` to this mapping 
![image](https://github.com/ccxt/ccxt/assets/43336371/0e123187-084f-41dd-b155-c1dbfda978f0)